x-pack/plugins/monitoring/public/views/cluster/listing/index.js

```diff
@@ -21,6 +22,12 @@ const getPageData = ($injector) => {
   return monitoringClusters(undefined, undefined, CODE_PATHS);
 };
 
+const getAlerts = (clusters) => {
+  const alerts = clusters.reduce((alerts, cluster) => ({ ...alerts, ...cluster.alerts.list }), {});
```

Comment:
@estermv How about directly returning the result and not assigning to a variable?